nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/PutDatabaseRecord.java

```diff
@@ -669,11 +685,20 @@ private void executeDML(ProcessContext context, ProcessSession session, FlowFile
                         }
                     }
                     ps.addBatch();
+                    if (++currentBatchSize == batchSize) {
```

Comment:
I'm not sure this would be benefitial. PutDatabaseRecord works without autoCommit. It's all or nothing.

Reply:
True, I missed that override before, but I see it now. So definitely less valuable, the only thing it would provide would be troubleshooting guidance, "your bad data is roughly in this part of the file". Probably not worth it. Thanks!